cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/CommandUtil.java

```diff
@@ -207,7 +196,14 @@ private static Path getTemplatePath() throws URISyntaxException {
      */
     public static void applyTemplate(Path modulePath, String template) throws IOException, URISyntaxException {
         Path templateDir = getTemplatePath().resolve(template);
-        Files.walkFileTree(templateDir, new FileUtils.Copy(templateDir, modulePath));
+        if (template.equalsIgnoreCase("main")) {
```

Comment:
This comment refers to L239. The existing template for lib contains the `observabilityIncluded` build option. We can remove the `[build-options]` section from the `manifest-lib.toml`